pkg/virt-launcher/virtwrap/api/defaults.go

```diff
@@ -16,17 +16,15 @@ type Defaulter struct {
 }
 
 func (d *Defaulter) IsPPC64() bool {
-	if d.Architecture == "ppc64le" {
-		return true
-	}
-	return false
+	return d.Architecture == "ppc64le"
```

Comment:
Please, if you refactor some code, add this in a separate commit

Reply:
Yes, it looks like i forgot that function when splitting the refactor into a separate commit. I added it to the separate PR i have open for that: https://github.com/kubevirt/kubevirt/pull/11602

I will leave it in here for now until i rebase again ontop of main, since that is easier.